src/api/app/views/webui/package/side_links/_checkout_package.html.haml

```diff
@@ -0,0 +1,8 @@
+%li
+  = link_to('#', aria: { controls: 'collapse-checkout', expanded: false }, 'data-toggle': 'collapse', href: '.collapse-checkout', role: 'button') do
```

Comment:
```suggestion
  = link_to('#', aria: { controls: 'collapse-checkout', expanded: false }, 'data-toggle': 'collapse', role: 'button') do
```

Reply:
Or button with data-target would probably the better choice. 

https://getbootstrap.com/docs/4.6/components/collapse/

Will also make the spider test like it.

Reply:
Just that diff did not work (was not expanding anymore), but thanks to your link to the docs I found something that works. Can it still be improved?